ai.py

```diff
@@ -1,98 +1,145 @@
 from typing import List, Tuple
-from board import Board, Move, Position
+from board import Board, Move, Position, diagonal_directions, file_directions, knight_directions
 import numpy
 
+def find_attacking_moves(board: Board, pos: Position, player_sign: None):
```

Comment:
Find moves from player player_sign that attack a given square (that's what this function does)